prov/efa/test/efa_unit_tests.h

```diff
@@ -8,9 +8,26 @@
 #include <setjmp.h>
 #include <cmocka.h>
 #include "stdio.h"
+#include "efa.h"
+#include "rxr.h"
+
+struct efa_resource {
+	struct fi_info* info;
+	struct fid_fabric* fabric;
+	struct fid_domain* domain;
+	struct fid_av* av;
+};
 
 void test_duplicate_efa_ah_creation();
 
 void test_efa_device_construct_error_handling();
 
+void test_rxr_ep_pkt_pool_flags();
+
+void test_rxr_ep_pkt_pool_page_alignment();
+
+int test_efa_resource_construct(struct efa_resource* resource);
```

Comment:
thinking about it. The struct name probably should  be `efa_unit_test_resource` and, function mame should be `efa_unit_test_resource_construct`. 

`test_xxx` is for a unit test.

Reply:
Fixed.